lib/was/registrar/register_object.rb

```diff
@@ -30,9 +30,9 @@ def register_object_using_web_service(register_params)
         Rails.logger.debug "Registering an object with params #{register_params}"
         begin
           resource = RestClient::Resource.new(
-            Rails.configuration.service_root, timeout: 300, open_timeout: 60
+            Rails.configuration.service_root, read_timeout: 300, open_timeout: 60
           )
-          response = resource.post(register_params)
+          response = resource.post(register_params, accept: :text)
```

Comment:
apparently the Sinatra app defaults to text, but the rails app defaults to json.  Because we want a druid as a string in the response, we now need this.